shims/common/src/main/scala/org/apache/gluten/config/GlutenConfig.scala

```diff
@@ -1776,6 +1778,13 @@ object GlutenConfig {
       .booleanConf
       .createOptional
 
+  val NOOP_WRITER_ENABLED =
+    buildConf("spark.gluten.sql.noop.writer.enabled")
```

Comment:
Nit:
`xxx.noopWriter.enabled`

Reply:
Curious why we need this config? Can we just use it for suitable case? If some Spark UT checks failed, we may modify it to adapt to Gluten's plan.

Reply:
> Curious why we need this config? Can we just use it for suitable case? If some Spark UT checks failed, we may modify it to adapt to Gluten's plan.

Currently just for suite cases.

Reply:
Would remove this config and fix suite tests.